### PR TITLE
Content/rates rebate research report

### PR DESCRIPTION
**Pull request by JacOng17**
Added the report in markdown as requested & added a link to that in the Rates Rebate project card.

### CODE REVIEW

File: _posts/2019-07-30-Rates-Rebate-Beta-Research-Report.md [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+### The results from this research inform how we might
+
+1. Help increase the uptake of the Rates Rebate
+2. Improve customer experience and satisfaction levels
+3. Decrease the cost of processing applications for councils
+4. Improve the ability to digitally deliver this and other services to councils
+5. Support the case to remove the statutory declaration (witnessed signature) requirement
+6. Reuse these insights for other use cases that have similar customer profiles
+
+This report was produced in parallel with the [Rates Rebates Beta Recommendations Report](https://docs.google.com/document/d/10JO2XS1g-PYD26S4P8FjSMD6Q3tJB72vnw55vuatqWQ/edit?usp=sharing) which documents the key results of the trial, presents options for next steps, and makes a set of recommendations.
```
Comment (gthurstonnz) [inline _posts/2019-07-30-Rates-Rebate-Beta-Research-Report.md]: Can we link this to the toolkit version of the report once it is up please

Comment (JacOng17) [reply _posts/2019-07-30-Rates-Rebate-Beta-Research-Report.md]: Sure, will change it :+1:

Comment (JacOng17) [inline _posts/2019-07-30-Rates-Rebate-Beta-Research-Report.md]:
```suggestion
This report was produced in parallel with the [Rates Rebates Beta Recommendations Report](2019/08/02/Rates-Rebates-Recommendations-Report/) which documents the key results of the trial, presents options for next steps, and makes a set of recommendations.
```

**Comment by JacOng17** [comment]
Done :smile:

**Pull request merged**